.github/workflows/node.js.yml

```diff
@@ -0,0 +1,70 @@
+name: fail_fast_CI
+on:
+  push:
+    branches:
+      - dev
+      - feature    
+      - ktchethan-patch-1
+  pull_request:
+  release:
+    types:
+      - staging
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  checkout_and_install:
+    strategy:
+      matrix:
+         node: ["15.14.0"]        
+        #node: ["12.22.1", "14.16.1", "15.14.0", "16.1.0"]        
+    runs-on: ubuntu-latest    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}          
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: npm ci      
+      - name: Installing nyc Istanbul
+        run: npm install nyc --save-dev
+      - name: npm build, and test
+        run: |
+          npm run build --if-present
+          npm test              
+      - name: Installing node_modules
+        run: npm install eslint --global
+      - name: Install Yarn
+        run: npm install -g yarn
+      - name: Run lint
+        run: yarn run eslint --init
+      - name: Installing coverage
+        run: npm install check-code-coverage            
+      - name: Upload coverage to Codecov
+        # finally run the tests
+        run: npm test      
```

Comment:
npm test is done totally 3 times across these two yml files. This consumes build time. Why is this needed?

Reply:
Accepted, will remove the instruction from node-js.yml.